pkg/kwok/controllers/pod_controller.go

```diff
@@ -416,8 +416,8 @@ func (c *PodController) WatchPods(ctx context.Context, lockChan chan<- *corev1.P
 					if c.needLockPod(pod) {
 						lockChan <- pod.DeepCopy()
 					} else {
-						logger.Info("Skip pod",
-							"reason", "not manage",
+						logger.Debug("Skip pod",
```

Comment:
Maybe we should change all the Skip/Delayed messages to debug instead of just this one

Reply:
previously I thought so too, but only changed this one when I saw others are logging under the managed nodes.

It is reasonable to change those skip logs, let me change them tonight